### ISSUE TITLE
'invalid_api_usage' error attempting to list subscriptions filtered by type

### ISSUE THREAD

**Issue by JimAllanson** [gocardless/gocardless-nodejs#83]
Attempting to set the `status` parameter on a SubscriptionListRequest results in an API error.

Steps:

```js
import { GoCardlessClient } from 'gocardless-nodejs/client';
import { SubscriptionStatus } from 'gocardless-nodejs/types/Types';

const gocardless = new GoCardlessClient(gocardlessKey, gocardlessEnv)
await gocardless.subscriptions.list({
  status: [SubscriptionStatus.Active],
});
```

Result:

```json
{
  "message": "One of your parameters was incorrectly typed",
  "errors": [
    {
      "field": "status",
      "message": "{\"0\"=>\"active\"} is not an array or string.",
      "request_pointer": "/subscriptions/status"
    }
  ],
  "documentationUrl": "https://developer.gocardless.com/api-reference#invalid_type",
  "type": "invalid_api_usage",
  "requestId": "<omitted>",
  "code": 422
}
```

I've also attempted to import `SubscriptionStatus` from `gocardless-nodejs` instead of `gocardless-nodejs/types/Types`, but SubscriptionStatus is undefined at run time when importing this way.

As a workaround when filtering by a single status, an @ts-ignore flag can be added to allow passing in a single string.

```
await gocardless.subscriptions.list({
  //@ts-ignore
  status: SubscriptionStatus.Active,
});
```

**Comment by tallosan** [comment]
Hi Jim,

Thanks for your patience. I'll look into this today and see if I can get it sorted for you.

**Comment by tallosan** [comment]
Should now be fixed in the latest version (`1.4.1`).
